webrender/res/cs_blur.glsl

```diff
@@ -5,131 +5,120 @@
 #include shared,prim_shared
 
 varying vec3 vUv;
-flat varying vec4 vUvRect;
-flat varying vec2 vOffsetScale;
-flat varying float vSigma;
-flat varying int vBlurRadius;
+flat varying vec2 vSrcSizeInv;
+flat varying vec4 vSrcRect;
+// The coefficient and `exp(2.0 * coefficient)`, in that order.
+flat varying vec2 vCoefficients;
+flat varying int vVertical;
+
+#ifdef WR_FEATURE_COLOR_TARGET
+#define TEXTURE_SIZE()  vec2(textureSize(sCacheRGBA8, 0).xy)
+#else
+#define TEXTURE_SIZE()  vec2(textureSize(sCacheA8, 0).xy)
+#endif
 
 #ifdef WR_VERTEX_SHADER
-// Applies a separable gaussian blur in one direction, as specified
-// by the dir field in the blur command.
-
-#define DIR_HORIZONTAL  0
-#define DIR_VERTICAL    1
 
 in int aBlurRenderTaskAddress;
 in int aBlurSourceTaskAddress;
-in int aBlurDirection;
 
 struct BlurTask {
     RenderTaskCommonData common_data;
-    float blur_radius;
+    vec2 coefficients;
+    int direction;
 };
 
-BlurTask fetch_blur_task(int address) {
+BlurTask fetchBlurTask(int address) {
```

Comment:
please stick to the existing naming convention in the shaders

Reply:
I made that change to improve consistency. Do function names have `snake_case` and variable names have CamelCase? I never was able to figure it out.

Reply:
Look at `prim_shared.glsl` for examples. It's pretty much the Rust naming convention:
  - variables and functions are `snake_case`
  - type names are `CamelCase`